### ISSUE TITLE
Fix accessibility issues flagged by Accessibility Tests

### ISSUE THREAD

**Issue by Velin92** [element-hq/element-x-ios#4336]
This is the first part of all the issues that were spotted by our accessibility tests that need to be fixed
 
- Placeholder views in the Home Screen did provide only gibberish unreadable labels, they should be unified in a single voice over object saying “Loading”
- (NOTE: this is part of the knocking redesign feature) when editing the room address (or creating one) since the row is split into the following 3 elements:
    - Text with the # symbol
    - Text field  where the user will input the address name
    - Text with the home server preceeded by : (e.g :[matrix.org](http://matrix.org/))
    - The non text field elements should be ignored and a hint should give away the full value
- Text fields blocks in the PIN screen do not have any descriptions  so the user might not have a clue of what they need to do when tapping over it
- The security key banner has a close button which has a hit area which is too small
- When adding a screenshot to the bug report screen, these are not labelled so they appear as voice over objects without any information
- The Home Screen cell when in placeholder should hide accessibility
- Hide accessibility on the avatar in the Join Room Screen
- Hit area for the warning button is too small in the media upload preview screen
- Mention Suggestions View should have its avatar accessibility be hidden, and the whole cell be grouped
- The message forwarding screen has empty rectangles that might be read by voice over as elements without any descriptions
- Notification Settings screen has a warning icon which has no description, and should be just hidden
- Read Receipt Cell should be grouped and the avatar accessibility should be hidden
- Room Change role selection for the users has
    - A too small hit area for the dismiss button
    - The avatar should not have an accessiibility label
    - We should group the whole element and add the dismiss action as an accessibility action
- The Room Selection screen has the issue as the message forwarding screen, which means empty rectangles are recognised by voice over and should be hidden
- Session verification request details is not grouped and avatar should have its accessibility hidden
- Thread summary view when in placeholder mode should have “loading” as accessibility label
- Thread summary view should also be grouped, the icon should be named thread and the avatar ignored
- Timeline Reply View should have accessibility hidden on the icon, and the whole reply view grouped together
- Knock request message should have its footer as a placeholder for voice over, and the count of characters read out as an accessibility hint
